tls/s2n_connection.c

```diff
@@ -222,6 +227,8 @@ int s2n_connection_wipe(struct s2n_connection *conn)
      */
     int mode = conn->mode;
     struct s2n_config *config = conn->config;
+    union s2n_prf_working_space prf_space = conn->prf_space;
```

Comment:
Per offline conversation: We're doing a ~300 byte copy of this struct twice. We really only need to preserve one field in the struct(The EVP handle). I think it makes sense to try and optimize this to only copy that handle.